src/library_async.js

```diff
@@ -108,12 +108,12 @@ mergeInto(LibraryManager.library, {
 #if ASYNCIFY_DEBUG
               err('asyncify: suspendOnReturnedPromise for', x, original);
 #endif
-              imports[x] = original = Asyncify.suspender.suspendOnReturnedPromise(
```

Comment:
Can/should we remove all references to `Asyncify.suspender`?

```
$ git grep Asyncify.suspender
src/library_async.js:      Asyncify.suspender = new WebAssembly.Suspender();
src/library_async.js:              imports[x] = original = Asyncify.suspender.suspendOnReturnedPromise(
src/library_async.js:            ret[x] = original = Asyncify.suspender.returnPromiseOnSuspend(original);
```

Reply:
Done